### PR TITLE
client/tailscale: update Client API a bit

### PR DESCRIPTION
**Pull request by bradfitz**

### CODE REVIEW

**Comment by bradfitz** [comment]
I have the corresponding change on the other side just about ready too. I won't submit this one until that one's also out & approved.

File: client/tailscale/tailscale.go
Comment (maisem) [inline client/tailscale/tailscale.go:82]: APIKey
AuthKey is used specifically for registration of nodes

File: client/tailscale/tailscale.go
Comment (maisem) [inline client/tailscale/tailscale.go:12]: Q?

File: client/tailscale/acl.go
Comment (maisem) [inline client/tailscale/acl.go:446]: It used to be APIKey too

**Review by maisem — APPROVED** [review]

File: client/tailscale/tailscale.go
Comment (bradfitz) [reply client/tailscale/tailscale.go:82]: braino

**Pull request merged**
